src/i18n/publishedProvider.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { PropsWithChildren } from "react";
+import { I18nextProvider } from "react-i18next";
+
+import i18n from "./i18n";
+import { PublishedProvider as LegacyProvider } from "./legacy";
+
+export default function PublishedProvider({ children }: PropsWithChildren<{}>) {
```

Comment:
Based on [this post](https://stackoverflow.com/a/71809927) by one of React's devs, he says PropsWithChildren doesn't need to be used, and is only useful for the `codemod`. 
Also, children typings are fixed in React 18, so we will need to start providing a prop type. Can you add that here? Type for children: `children?: React.ReactNode`